continuous_integration/environment-3.8-jdk11-dev.yaml

```diff
@@ -3,41 +3,41 @@ channels:
 - conda-forge
 - nodefaults
 dependencies:
-- adagio>=0.2.3
-- antlr4-python3-runtime>=4.9.2, <4.10.0 # Remove max pin after qpd(fugue dependency) updates their conda recipe
-- black=22.3.0
-- ciso8601>=2.2.0
-- dask-ml>=2022.1.22
-- dask>=2022.3.0
-- fastapi>=0.61.1
-- fs>=2.4.11
-- intake>=0.6.0
-- isort=5.7.0
-- jpype1>=1.0.2
+- dask-ml=2022.1.22
+- dask=2022.3.0
+- fastapi=0.69.0
+- intake=0.6.0
+- jpype1=1.0.2
 - jsonschema
-- lightgbm>=3.2.1
-- maven>=3.6.0
-- mlflow>=1.19.0
-- mock>=4.0.3
-- nest-asyncio>=1.4.3
+- lightgbm
+- maven
+- mlflow<1.26.1  # remove constraint when Windows mlflow issues are resolved
```

Comment:
Hmm I was under the impression that this was a breaking change introduced in MLflow 1.26.1, but other passing tests indicate that this might be just an issue with this PR due to the pinned dependencies?

I will remove this constraint to see if the failures are still occurring.

Reply:
Failures seem to be tied to `protobuf`, which I notice we're pulling in an older version of with Arrow pinned to 1.0.1. I've updated our Arrow min version to 3.0.0, which should allow us to pull in the latest version of `protobuf` and hopefully resolve.

In any case, I notice that MLflow 1.26.1 constrains `protobuf >=3.7.0`, which might be insufficient for Windows given the failures here - I'll try getting a reproducer together to open up an issue there.